applications/ChimeraApplication/chimera_application_variables.h

```diff
@@ -0,0 +1,38 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+// ==============================================================================
+//  ChimeraApplication
+//
+//  License:         BSD License
+//                   license: ChimeraApplication/license.txt
+//
+//  Authors:        Aditya Ghantasala, https://github.com/adityaghantasala
+// 					Navaneeth K Narayanan
+//					Rishith Ellath Meethal
+// ==============================================================================
+//
+
+#if !defined(KRATOS_CHIMERA_APPLICATION_VARIABLES_H_INCLUDED)
+#define KRATOS_CHIMERA_APPLICATION_VARIABLES_H_INCLUDED
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/define.h"
+#include "includes/variables.h"
+
+namespace Kratos
+{
+
+KRATOS_DEFINE_FLAG( FS_CHIMERA_VEL_CONSTRAINT);
+KRATOS_DEFINE_FLAG( FS_CHIMERA_PRE_CONSTRAINT);
```

Comment:
I think it's worth it to write the whole word
```suggestion
KRATOS_DEFINE_FLAG( FS_CHIMERA_PRESSURE_CONSTRAINT);
```